sw/airborne/modules/obstacle_avoidance/obstacle_avoidance.c

```diff
@@ -52,175 +53,146 @@
 #ifndef AVOIDANCE_WIDTH_IN_MEASUREMENT_VALUES
 #define AVOIDANCE_WIDTH_IN_MEASUREMENT_VALUES 6
 #endif
-//////////////SET BY USER!!!!//////////////////
-//sensor info
-uint16_t size_matrix[] = {AVOIDANCE_AMOUNT_OF_BOARDS, AVOIDANCE_HEIGHT_IN_MEASUREMENT_VALUES, AVOIDANCE_WIDTH_IN_MEASUREMENT_VALUES};
-float stereo_fow[2] = {1.0018, 0.7767};//based on FOW of 57.4, by 44.5
-float angle_hor_board[] = {0, 1.0472, 2.0944, 3.1416, -2.0944, -1.0472};
-int16_t focal = 118 * 6;
-float baseline = 60;
-float ref_yaw = 0.0;
-//tuning info
-float reference_pitch = 0.1;
-float reference_roll = 0.1;
-float dist_treshold = 1.25;
-float distances_hor[AVOIDANCES_DISTANCES_HOR_COUNT];
 
-//////////////Variables CN///////////////////
-//for messages
-//general
-uint8_t *READimageBuffer_old; //USED for butterworth filter
-float heading_goal_f = 0;
-float heading_goal_ref;
-struct FloatVect2 pos_diff;
-//target
-struct NedCoor_f current_pos;
-struct FloatVect2 target;
-struct FloatVect2 init_target = {0, 0};
+//#ifndef PRINT_MATRIX
+//#define PRINT_MATRIX
+//#endif
+
+//#ifndef SEND_OA_DATA
+//#define SEND_OA_DATA
+//#endif
+
+//Initialize variables which can be changed in settings
+//OA_general
 float dx_ref = 0;
 float dy_ref = 0;
-//Variables Kalman filter
-float A_kal = 1;
-float B_kal = 0;
-float H_kal = 1;
-float Q_kal = 0.05;//TODO: clean variables
-float R_kal = 2;
-float K_gain_send = 0;
-float Pest_new[AVOIDANCE_AMOUNT_OF_BOARDS *AVOIDANCE_HEIGHT_IN_MEASUREMENT_VALUES
-               *AVOIDANCE_WIDTH_IN_MEASUREMENT_VALUES];
-float Xest_new[AVOIDANCE_AMOUNT_OF_BOARDS *AVOIDANCE_HEIGHT_IN_MEASUREMENT_VALUES
-               *AVOIDANCE_WIDTH_IN_MEASUREMENT_VALUES];
-//variables butterworth filter
-struct FloatVect3 filter_repforce_old = {0.0, 0.0, 0.0};
-struct FloatVect3 Repulsionforce_Kan_old = {0.0, 0.0, 0.0};
-float butter_old[AVOIDANCE_AMOUNT_OF_BOARDS *AVOIDANCE_HEIGHT_IN_MEASUREMENT_VALUES
-                 *AVOIDANCE_WIDTH_IN_MEASUREMENT_VALUES];
-//Vector Method
+//Vector_method
 float F1 = 0.1;
 float F2 = 0.9;
 float Cfreq = 1;
-float Ko = 60;//1000;//10000
-float Kg = 100;//100;//100
-float Dist_offset = 0;//-0.125;
+float Ko = 60;
+float Kg = 100;
+float Dist_offset = 0;
 int8_t dis_treshold = 2;
-struct FloatVect3 Repulsionforce_Used;
-struct FloatVect3 Repulsionforce_Kan;
-struct FloatVect3 Attractforce_goal;
-//Potential Method
-float  b_damp = 0;
-float  K_goal = 1;
+//Pot_method
 float  K_obst = 40;
+float  K_goal = 1;
+float  b_damp = 0;
 float  c1_oa = 0.4;
 float  c2_oa = 0.4;
 float  c3_oa = 5.0;
 float  c4_oa = 0.5;
 float  c5_oa = 0.9;
```

Comment:
More descriptive variable names (put them together in a struct would also be fine.